actions/pacstrap_action.go

```diff
@@ -81,29 +81,13 @@ func (d *PacstrapAction) Run(context *debos.DebosContext) error {
 	}
 	f.Close()
 
-	// Create base layout for pacman-key
-	err = os.MkdirAll(path.Join(context.Rootdir, "var", "lib", "pacman"), 0755)
-	if err != nil {
-		return fmt.Errorf("Couldn't create var/lib/pacman in image: %v", err)
-	}
-	err = os.MkdirAll(path.Join(context.Rootdir, "etc", "pacman.d", "gnupg"), 0755)
-	if err != nil {
-		return fmt.Errorf("Couldn't create etc/pacman.d/gnupg in image: %v", err)
-	}
-
 	// Run pacman-key
 	cmdline := []string{"pacman-key", "--nocolor", "--config", configPath, "--init"}
```

Comment:
There's a conflict here between re-using the host's pacman configuration and making it work in fakemachine (without root privileges). If you don't have root privileges, you can't read some of the GPG configuration for the host, that means you need to make an unprivileged GPG configuration. It's probably possible (but really messy I think?) to detect whether you have the necessary privileges, but I got around that in a subsequent commit by making the public keyring available even when creating a new pacman configuration.